u8timeseries/models/forecasting_model.py

```diff
@@ -63,6 +66,9 @@ def predict(self, n: int) -> TimeSeries:
         if (not self._fit_called):
             raise_log(Exception('fit() must be called before predict()'), logger)
 
+    def get_min_train_series_length(self) -> int:
```

Comment:
How about using a `@property`? `base_class.min_series_length` and `sub_class.min_series_length` seems a better approach. It makes it clearer what it represents, i.e. a property of ForecastingModel's

Reply:
And if it needs to be specific to each subclass, could raise NotImplementedError in base

Reply:
I like the property decorator idea, I will implement that. In regards to your second comment, I thought it might be useful to have a default value, so that models that do not have a specific requirement can just inherit the default.